include/boost/json/impl/value_stack.ipp

```diff
@@ -364,16 +364,25 @@ push_array(std::size_t n)
     st_.exchange(std::move(ua));
 }
 
-void
+error_code
```

Comment:
this is going to force the construction of a new error code for every object even when there is no error and even when ignore_duplicates is true. It should be an outparam. Or even a bool